packages/prediction-app/src/components/ErrorPage.tsx

```diff
@@ -1,14 +1,15 @@
 import React from 'react'
+import i18n from '@dhis2/d2-i18n'
 import { useRouteError } from 'react-router-dom'
 
 const ErrorPage = () => {
-    const error = useRouteError()
+    const error = useRouteError() as { statusText?: string; message?: string }
```

Comment:
you can use `isRouteErrorResponse` from `react-router-dom` narrow the type instead of type-assertion